### PR TITLE
Create 2022-05-10-shuttleworth.md

### PR DESCRIPTION
**Pull request by kasiaocf**
# Pull Request

## Description

Add shuttleworth blog

Fixes #

## How Has This Been Tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration_

- [ ] Yes

_If your changes affect data processing, have you plotted any changes? i.e. have you done a quick sanity check?_

- [ ] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by peterdudfield** [comment]
Text all good, 
I think we dont need to worry about 'ocf-website' not working. 

I'll just looking into markdown showing an image, as it seemed to not show on[ tempt link](https://old-website-old-git-blog-shuttleworth-openclimatefix.vercel.app/blog/2022-05-10-shuttleworth)

**Comment by peterdudfield** [comment]
@kasiaocf could you rename the file to fit in with the other blog posts? i.e remove the '<' and '>'

File: src/posts/<2022-05-10>-<shuttleworth>.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+## Our Partner: The Shuttleworth Foundation
+Back in November 2021, one of our community members, Aaron, nominated us for a Flash Grant with the Shuttleworth Foundation. Twice a year the foundation awards a number of small grants to a collection of social change agents in support of their work.
+
+Openly publishing so many different ML model architectures over the last six months, as explained in this blog post, would not have been possible without the fantastic support from the Shuttleworth Foundation. Thank you so much, Aaron and Jason, your support has been incredibly helpful!
+
+We are planning to release more datasets in the future and engage with our community and produce more models that are useful for a wider audience. We will also publish pretrained models soon. Stay tuned!
+
+<img src="https://github.com/openclimatefix/website/blob/main/src/images/logos/shuttleworth.jpg" width="500" height="200" class="centerImage">
```
Comment (peterdudfield) [inline src/posts/<2022-05-10>-<shuttleworth>.md]: I think something like this would work
```
![shuttleworth](../images/logos/shuttleworth.jpg)
```

**Review by peterdudfield — CHANGES_REQUESTED** [review]
Thanks @kasiaocf for putting this together

- fix filename
- fix image display

File: src/posts/<2022-05-10>-<shuttleworth>.md (hunk 1 above)
Comment (kasiaocf) [reply src/posts/<2022-05-10>-<shuttleworth>.md]: let me try

**Comment by peterdudfield** [comment]
@all-contributors please add @kasiaocf for blog

**Review by peterdudfield — APPROVED** [review]

**Pull request merged**